lib/tario/compare.go

```diff
@@ -77,10 +73,7 @@ func isSimilarHardLink(h *tar.Header, nh *tar.Header) (bool, error) {
 // isSimilarDirectory returns if the given headers are describing similar
 // directories. It only checks mtime and owner, ignoring size, path and content.
 func isSimilarDirectory(h *tar.Header, nh *tar.Header) (bool, error) {
-	hMtime := h.ModTime.Truncate(1 * time.Second)
```

Comment:
We certainly need to check mtime for normal cases。
You can add parameter both to these functions and to the command line to disable mtime check.